gateway-ha/src/main/java/io/trino/gateway/proxyserver/ProxyRequestHandler.java

```diff
@@ -287,26 +286,26 @@ private FluentFuture<ProxyResponse> executeHttp(Request request)
     private ProxyResponse recordBackendForQueryId(Request request, ProxyResponse response, Optional<String> username,
             RoutingDestination routingDestination)
     {
-        log.debug("For Request [%s] got Response [%s]", request.getUri(), response.body());
+        log.debug("For Request [%s] got Response [%s]", request.getUri(), response.getDecompressedBody());
```

Comment:
This method has several `getDecompressedBody` usage, and the method decompresses every call. I don't we should do that.